.github/fixtures/test-release-statistics/cliff.toml

```diff
@@ -0,0 +1,48 @@
+# git-cliff ~ configuration file
+# https://git-cliff.org/docs/configuration
+
+[changelog]
+# A Tera template to be rendered for each release in the changelog.
+# See https://keats.github.io/tera/docs/#introduction
+body = """
+{% if version %}\
+    ## [{{ version | trim_start_matches(pat="v") }}] - {{ timestamp | date(format="%Y-%m-%d") }}
+{% else %}\
+    ## [unreleased]
+{% endif %}\
+{% for group, commits in commits | group_by(attribute="group") %}
+    ### {{ group | upper_first }}
+    {% for commit in commits %}
+        - {{ commit.message | upper_first }}\
+    {% endfor %}
+{% endfor %}
+### Commit Statistics
+* {{ statistics.commit_count }} commit(s) contributed to the release.
+{%- if statistics.commits_timespan is defined %}
+	* {{ statistics.commits_timespan }} day(s) passed between the first and last commit.
+{%- endif %}
+* {{ statistics.conventional_commit_count }} commit(s) {% if statistics.conventional_commit_count > 1 %}were{% else %}was{% endif %} parsed as conventional.
+* {{ statistics.links | length }} issue(s) like '(#ID)' {% if statistics.links | length != 1 %}were{% else %}was{% endif %} seen in commit messages.
```

Comment:
This is too cumbersome... I think we can simplify this message/template.

An example:

Before:

> 3 issue(s) like '(#ID)' were seen in commit messages.

After:

> 3 linked issue(s) detected in commits.

WDYT?

Reply:
Agreed, it's a bit redundant. I'll simplify the message and remove the conditional logic.